src/game/default/g_cmd.c

```diff
@@ -942,12 +1012,13 @@ static void G_Ready_f(g_entity_t *ent) {
 	}
 
 	if (ent->client->locals.persistent.ready) {
-		gi.ClientPrint(ent, PRINT_HIGH, "You're already ready\n");
+		G_Unready_f(ent);
```

Comment:
Yes, I changed this to a toggle. Perhaps a better way would be to have separate **ready** and **unready** commands and add a **toggleready** command. I find it handy to have a single key bound to ready/unready and toggling is very handy. 

I also often forget I've readied up and then unready which messes with all sorts of other complicated aliases and causes me to swear loudly. Part of the fun :)